src/tests/product_soring.spec.js

```diff
@@ -0,0 +1,18 @@
+import { expect } from '@playwright/test';
+import test from '../fixtures/base';
+import { CUSTOMERS } from '../constants/users';
+
+test.describe('Saucedemo app basic tests', () => {
+    const standartUserCredentials = CUSTOMERS[0];
+    test('Check to sort products by name (Z to A)', async (
+        /** @type {{ app: import('../pages/Application').Application }} */{ app },
+    ) => {
+        await app.login.navigate();
+        await app.login.performLogin(standartUserCredentials.userName, standartUserCredentials.password);
+        const namesBeforeSort = await app.inventory.getProductNames();
+        await app.inventory.sortBy('za');
```

Comment:
You should implement all available options for sorting:
- Name (A to Z)
- Name (Z to A)
- Price (low to high)
- Price (high to low)